public/components/_drinks.jade

```diff
@@ -6,4 +6,6 @@ div.jsla-content-section.layer-2.jsla-drinks-container
         h1 Drinks.js
         h4 after the talks, around 9:15
         if url
-          h3.jsla-drinks-location!= "at <a href='" + url + "' title='" + name + "' target='_blank'>" + name + "</a>"
```

Comment:
This PR addresses an issue where a location name with a single quote would break harp. It's related to this html concatenation.